netsync/manager.go

```diff
@@ -189,7 +189,7 @@ func (sm *SyncManager) startSync() {
 		log.Infof("Syncing to block %s from peer %s",
 			bestPeer.SelectedTip(), bestPeer.Addr())
 
-		bestPeer.PushGetBlockLocatorMsg(&daghash.ZeroHash, sm.dagParams.GenesisHash)
+		bestPeer.PushGetBlockLocatorMsg(bestPeer.SelectedTip(), sm.dagParams.GenesisHash)
```

Comment:
While you are there refactoring stuff, how about rename bestPeer to syncPeer, as it's called anywhere else?